src/main/java/com/iemr/tm/utils/CookieUtil.java

```diff
@@ -0,0 +1,41 @@
+package com.iemr.tm.utils;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+@Service
+public class CookieUtil {
+
+	public Optional<String> getCookieValue(HttpServletRequest request, String cookieName) {
+		Cookie[] cookies = request.getCookies();
+		if (cookies != null) {
+			for (Cookie cookie : cookies) {
+				if (cookieName.equals(cookie.getName())) {
+					return Optional.of(cookie.getValue());
+				}
+			}
+		}
+		return Optional.empty();
+	}
+
+	public void addJwtTokenToCookie(String Jwttoken, HttpServletResponse response) {
```

Comment:
Why is this method present in every service line?
Isn't this cookie set on login? When does it get generated again from service lines?

Reply:
fixed

Reply:
Sorry, fixed what?